draft/security.md

```diff
@@ -0,0 +1,109 @@
+
+We welcome contributors with various levels of experience, but due to the sensitive nature of security work, members should demonstrate a track record of responsible security practices. Experience with security vulnerability assessment, analysis, or remediation is a plus.
+
+### Access granting and deployment
+
+Initial onboarding phase where only Chair and Co-Chair have permissions to access security reports and coordinate responses. After onboarding of 6 months, other group members who have proven their expertise will be granted access to the security issue tracker and response coordination at the discretion of the chair and co-chair.
+
+The deployment of security fixes will be coordinated with the Django release team and the ops team.
+
+### How do people who want to join sign up / volunteer / express interest?
+
+Individuals can express interest by opening a PR to this working group repository using a template, adding their names in the list of members along with a brief description of their security experience and motivation to join the group.
+
+Alternatively, individuals can send an email to the Chair or Co-Chair expressing interest and outlining their qualifications and motivation.
+
+### How will decisions on adding/removing members be handled?
+
+Direct membership: new members may self-nominate; the WG will vote (50%+1) to approve/deny new members. The WG will vote for New Chair/Co-Chairs and decision to appoint will be based on gaining majority votes.
+
+Members join the group for one year term. At the end of this term, they need to opt into staying involved to keep being a member of the group.
+
+Due to the sensitive nature of security work, members who fail to adhere to responsible disclosure practices or exhibit other concerning behavior may be removed by a vote of the working group (50%+1) or by decision of the Chair and Co-Chair in consultation with the Board Liaison.
+
+## Budget
+
+No allocated budget.
+
+Sums may be provided for specific activities (e.g. security audits, external consultancy, security tooling, bug bounty programs) subject to approval by the DSF board.
+
+## Communications
+
+- Private channel in the DSF slack for internal discussions
+- Encrypted communication channels for sensitive security discussions
+- Public reports of resolved security issues on the Django security page
+- The Django forum sub-section "Security" of Django Internals for general security discussions
+
+## Reporting
+
+We'll maintain a private log of all security reports and their resolutions. Public disclosures will follow Django's established security release process.
```

Comment:
There's an email history from the list, but otherwise I don't think there's a "log" of reports (besides ones which are published).